src/Nventive.Persistence.Sample/Nventive.Persistence.Sample.Mobile/Android/AndroidManifest.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.companyname.Nventive.Persistence.Sample" android:versionCode="1" android:versionName="1.0">
+  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="31" />
```

Comment:
We can update the sample min sdk versions to 26 (Android 8)

Reply:
What about iOS?